pkg/cli/start/start.go

```diff
@@ -141,7 +161,7 @@ func (l *LoftStarter) prepare() error {
 		contextToLoad = l.Context
 	} else if platformConfig.LastInstallContext != "" && platformConfig.LastInstallContext != contextToLoad {
 		contextToLoad, err = l.Log.Question(&survey.QuestionOptions{
-			Question:     product.Replace("Seems like you try to use 'loft start' with a different kubernetes context than before. Please choose which kubernetes context you want to use"),
+			Question:     product.Replace(fmt.Sprintf("Seems like you try to use 'loft %s' with a different kubernetes context than before. Please choose which kubernetes context you want to use", l.CommandName)),
```

Comment:
Can we also rename the command from `'loft %s'` to `'vcluster platform %s'` here?

Reply:
will fix!